cmd/kops/delete_cluster.go

```diff
@@ -215,3 +218,16 @@ func RunDeleteCluster(f *util.Factory, out io.Writer, options *DeleteClusterOpti
 	fmt.Fprintf(out, "\nCluster deleted\n")
 	return nil
 }
+
+func filterUserProvidedSubnets(resourceTrackers map[string]*resources.ResourceTracker, cluster *api.Cluster) {
```

Comment:
Can we move this into the resources/delete? We are trying to separate the specific cloud stuff into there own homes.